drivers/wifi/siwx91x/siwx91x_wifi.c

```diff
@@ -138,6 +180,7 @@ static int siwx91x_ap_enable(const struct device *dev, struct wifi_connect_req_p
 	struct siwx91x_dev *sidev = dev->data;
 	/* Wiseconnect requires a valid PSK even if WIFI_SECURITY_TYPE_NONE is selected */
 	static const char dummy_psk[] = "dummy_value";
+	sl_wifi_ap_configuration_t saved_ap_cfg = { };
```

Comment:
This variable is going to be overwritten by `get_saved_ap_configuration()`, so it is not required to initialize it.